examples/exclude-words/synthesizer.py

```diff
@@ -12,6 +12,7 @@ class ExcludeWordsSynthesizer(BasicSynthesizer):
     """
     WORDS_DELIM = ' '
     WORDS_FILE = pathlib.Path(__file__).parent / 'words.json'
+    CAPITALIZE_WORDS = True
```

Comment:
Is this needed? Or do we always want this behaviour. It could be misunderstood to mean that we capitalize the words on the invoice.

Reply:
Agree that the variable name might be confusing. Will try to come up with a better one, any ideas?

The behavior is not always wanted imo and I wanted to make it easy to turn ON/OFF here since they are just examples of using the library.

Reply:
How about `IGNORE_CAPS` instead?